src/uno/RoslynQuoter.Shared/UnoPlatformShell.xaml

```diff
@@ -0,0 +1,327 @@
+
+								<!-- App Name -->
+								<TextBlock Text="{TemplateBinding AppName}"
+										   Foreground="{StaticResource Color02Brush}"
+										   FontSize="22"
+										   FontWeight="Bold" />
+
+								<!-- by -->
+								<TextBlock Text=" by "
+										   Foreground="{StaticResource Color02Brush}"
+										   FontSize="22">
+								</TextBlock>
+
+								<!-- App Author -->
+								<TextBlock Text="{TemplateBinding AppAuthor}"
+										   Foreground="{StaticResource Color02Brush}"
+										   FontSize="22" />
+							</StackPanel>
+
+							<!-- About Button -->
+							<Button x:Name="openAboutButton"  
+									Content="ABOUT"
+									Style="{StaticResource PrimaryButtonStyle}"
+									Click="OnAbout"
+									VerticalAlignment="Center"
+									Margin="0,0,20,0"
+									Grid.Column="2" />
+						</Grid>
+
+						<!-- App Content -->
+						<ContentPresenter x:Name="ContentPresenter"
+										  Content="{TemplateBinding AppContent}"
+										  Grid.Row="1" />
+
+						<!-- About Popup -->
+						<Grid Visibility="{TemplateBinding AboutVisibility}"
+							  Grid.Row="0"
+							  Grid.RowSpan="2">
+
+							<!-- Soft Dismiss Button -->
+							<Button x:Name="softDismissAboutButton"
+									Style="{StaticResource BasicEmptyButtonStyle}"
+									Click="OnCloseAbout" />
+
+							<!-- Popup -->
+							<Grid Background="{StaticResource Color01Brush}"
+								  CornerRadius="4"
+								  MaxWidth="580"
+								  HorizontalAlignment="Center"
+								  VerticalAlignment="Top"
+								  Margin="0,170,0,0">
+
+								<!-- Close Button -->
+								<Button x:Name="closeAboutButton"
+										Style="{StaticResource BasicEmptyButtonStyle}"
+										Click="OnCloseAbout"
+										VerticalAlignment="Top"
+										HorizontalAlignment="Right"
+										Padding="0">
+
+									<!-- Icon -->
+									<Path Data="M14 1.41 12.59 0 7 5.59 1.41 0 0 1.41 5.59 7 0 12.59 1.41 14 7 8.41 12.59 14 14 12.59 8.41 7"
+										  Fill="{StaticResource Color02Brush}"
+										  Stretch="Uniform"
+										  Height="14"
+										  Width="14"
+										  Margin="16" />
+								</Button>
+
+								<StackPanel Margin="40,40,40,65">
+
+									<!-- Header -->
+									<TextBlock TextWrapping="Wrap"
+											   Foreground="{StaticResource Color02Brush}"
+											   FontWeight="Bold"
+											   FontSize="24">
+					
+										Welcome to the 
+					
+										<!-- Link to Uno Fork -->
+										<Hyperlink NavigateUri="{TemplateBinding LinkToUnoPlatformlApp}"
+												   Foreground="{StaticResource Color02Brush}">
+											<Underline>
+												Uno Platform Fork
+											</Underline>
+										</Hyperlink> 
+					
+										of Kirill Osenkov's 
+					
+										<!-- Link to Original Fork -->
+										<Hyperlink NavigateUri="{TemplateBinding LinkToOriginalApp}"
+												   Foreground="{StaticResource Color02Brush}">
+											<Underline>
+												Roslyn Quoter App

```

Comment:
is it expected to have Roslyn Quoter App in the Style ? I think it's meant to be reusable.

Reply:
No, my bad I used template binding for the links but forgot the the author name and project

Reply:
Done.